bigbluebutton-html5/imports/ui/services/api/index.js

```diff
@@ -36,7 +36,7 @@ function makeCall(name, ...args) {
  */
 function call(name, ...args) {
   return makeCall(name, ...args).catch((e) => {
-    NotificationService.add({ notification: `Error while executing ${name}` });
+    notify(`Ops! Error while executing ${name}`, 'error');
```

Comment:
So are we making it a console log instead?

Reply:
Don't know. I just changed from the old service to the new one.

Reply:
I think it should be a toast

Reply:
I like it being a Toast instead of a `console.error` because then the user can feedback us with "Hey I was trying to do X and the client showed Error while executing endMeeting".

But we should log that to the server too.